### PR TITLE
[NPM] [bug] Adding sort in iptable comment for deterministic behavior in Flake UT

### PR DESCRIPTION
**Pull request by vakalapa**
TestAllowMultiplePodSelectors has a flake UT result due to ordering of ipset name in the comment. This PR fixes it

### CODE REVIEW

File: npm/translatePolicy.go [hunk 1]
```diff
@@ -200,11 +202,13 @@ func craftPartialIptablesCommentFromSelector(ns string, selector *metav1.LabelSe
...
-	for i, _ := range labelsWithoutOps {
-		comment += prefix + ops[i] + labelsWithoutOps[i]
-		comment += "-AND-"
+	comments := []string{}
+	for i := range labelsWithoutOps {
+		comments = append(comments, prefix+ops[i]+labelsWithoutOps[i])
 	}
 
+	sort.Strings(comments)
+	comment := strings.Join(comments, "-AND-") + "-AND-"
```
Comment (rbtr) [inline npm/translatePolicy.go:211]: are you appending `-AND-` but then immediately chomping it off below when returning `[:len - len("-AND-")]`?

**Review by rbtr — COMMENTED** [review]
lgtm with one question/change

File: npm/translatePolicy.go (hunk 1 above)
Comment (vakalapa) [reply npm/translatePolicy.go]: Correcting that.

**Review by rbtr — APPROVED** [review]
lgtm

**Pull request merged**